app/modules/cinema/endpoints_cinema.py

```diff
@@ -37,6 +44,27 @@
 hyperion_error_logger = logging.getLogger("hyperion.error")
 
 
+@module.router.get(
+    "/cinema/movie/{movie_id}",
+    response_model=schemas_cinema.Movie,
+)
+def get_movie(movie_id):
+    url = f"https://api.themoviedb.org/3/movie/{movie_id}?api_key={API_key}&language=fr-FR"
+    try:
+        r = get(url, timeout=5)
+        response = loads(r.content)
+        return schemas_cinema.Movie(
+            genres=response["genres"],
+            overview=response["overview"],
+            poster_path=response["poster_path"],
+            title=response["title"],
+            runtime=response["runtime"],
+            tagline=response["tagline"],
+        )
+    except RequestException as error:
+        raise HTTPException(status_code=504, detail=str(error))
```

Comment:
If another error happens, you may want to log it but return a generic error message to Titan. Sensitive data could indeed be included in the error by themoviedb.
An error 500 could maybe be more adapted.